brigadier/projects/templates/task_list_included.html

```diff
@@ -0,0 +1,51 @@
+{% load i18n %}
+
+{% if task_list %}
```

Comment:
this check should be done before including a template

Reply:
Why? In project detail view we could face with same situation when we haven't any task in the project.